http/http-api/src/main/java/com/okta/commons/http/config/HttpClientConfiguration.java

```diff
@@ -141,6 +146,16 @@ public HttpClientConfiguration setRetryMaxAttempts(int retryMaxAttempts) {
         return this;
     }
 
+    public void setRequestExecutorParams(Map<String, String> map) {
+        if (map != null) {
```

Comment:
`map` can never be null here (unnecessary check). You're instantiating it in line #42.

Reply:
fixed